src/main/java/dumshenko/daniil/Shape.java

```diff
@@ -0,0 +1,26 @@
+package dumshenko.daniil;
+
+enum FigureType {
+    CIRCLE,
+    RECTANGLE,
+    SQUARE
+}
+
+public class Shape {
+    FigureType typeOfFigure;
```

Comment:
Ultra nit: Feels a bit more natural to me to just call this var `figureType`.
```suggestion
    FigureType figureType;
```